drivers/photo-electronic/adpd410x/iio_adpd410x.c

```diff
@@ -84,6 +84,41 @@ static int adpd410x_iio_read_raw_chan(void *device, char *buf, uint32_t len,
 	return snprintf(buf, len, "%d", data[channel->ch_num]);
 }
 
+/**
+ * @brief Read ADC data samples .
+ * @param device - Device driver descriptor.
+ * @param buff - Input buffer.
+ * @param nb_samples - Input number of samples.
+ * @return Number of samples, or negative code.
+ */
+static int32_t adpd410x_read_samples(void *device, uint32_t *buff,
+				     uint32_t nb_samples)
+{
+	struct adpd410x_dev *dev = (struct adpd410x_dev *)device;
+
+	int32_t ret;
+	uint32_t data[8], i, offset = 0;
```

Comment:
nitpick: could we use a constant for 8 here and use it in all other places? Will make it easier to maintain in the future.

Reply:
sure, I can add a commit prior to this one, with the suggested change.